t/perf/p2000-sparse-operations.sh

```diff
@@ -112,6 +112,7 @@ test_perf_on_all git commit -a -m A
 test_perf_on_all git checkout -f -
 test_perf_on_all git reset
 test_perf_on_all git reset --hard
+test_perf_on_all git read-tree -mu HEAD
```

Comment:
If you reorder this to be early in the series, you can then advertise the results of this test in the commit message for the commit that improves the performance.

Reply:
I moved the commit, but haven't added performance numbers yet (like with the `git status` change, I'll add those once any outstanding bugs are fixed). In the meantime, I'll keep this open to remind myself.